testData/solutions/win32.json5

```diff
@@ -2609,495 +2834,13473 @@
+                            "enum": [
+                                0,
+                                1,
+                                2,
+                                3
+                            ],
+                            "enumLabels": [
+                                "Turn Off",
+                                "Indicate with Sound",
+                                "Speak Count",
+                                "Describe Inconsistences"
+                            ],
+                            "default": 0
+                        }
+                    },
+                    "Options.TextAnalyserTypeFlags": {
+                        "schema": {
+                            "title": "Text Analyser Type",
+                            "description": "Indicates how the text analyzer should threat different symbols.",
+                            "type": "integer"
+                        },
+                        "label": "bitmask"
+                    },
+                    "Options.TextAnalyserSymbolFlags": {
+                        "schema": {
+                            "title": "Text Analyser Symbols",
+                            "description": "Indicates the symbols the text analyzer should check.",
+                            "type": "integer"
+                        },
+                        "label": "bitmask"
+                    },
+
+                    // ------------------ Convenient OCR ------------------
+
+                    // NOTE: The language codes used to describe the primary and secondary languages
+                    // are taken from the following list:
+                    //
+                    //  -https://docs.microsoft.com/es-es/windows/win32/intl/language-identifier-constants-and-strings
+                    //
+
+                    "OCR.PrimaryRecognitionLanguage": {
+                        "schema": {
+                            "title": "Primary OCR Recognition Language",
+                            "description": "Indicates the primary language for OCR.",
+                            "type": "integer",
+                            "default": 1033
+                        }
+                    },
+                    "OCR.SecondaryRecognitionLanguage": {
+                        "schema": {
+                            "title": "Secondary OCR Recognition Language",
+                            "description": "Indicates the secondary language for OCR.",
+                            "type": "integer",
+                            "default": 1033
+                        }
+                    },
+
+                    // --------------------- Gestures -----------------------
+
+                    "Touch.TapEventDurationMax": {
+                        "schema": {
+                            "title": "Explore Delay",
+                            "description": "Sets the maximum duration of the tapping event.",
```

Comment:
This sentence could be clearer.  Unless we're monitoring seances for spectral tapping, I assume "duration of the tapping event" is better expressed as something like "duration of a touchscreen press that will be considered as a 'tap'".

Reply:
Hahahha, good recommendation. I've changed the wording to yours.